app/_layouts/extension.html

```diff
@@ -216,13 +304,47 @@
 
 {{ plugin_config_for_route_admin_api }}
 
+`<route>` is the `id` or `name` of the Route that this plugin configuration
+  will target.
+
 {% else %}
 {% navtabs %}
 {% navtab Admin API %}
 
 {{ plugin_config_for_route_admin_api }}
 
+`<route>` is the `id` or `name` of the Route that this plugin configuration
+  will target.
+
 {% endnavtab %}
+{% unless page.params.manager_examples == false %}
+{% navtab Kong Manager %}
+Configure this plugin on a Route:
+
+1. In Kong Manager, select the workspace.
+2. From the Dashboard, select **Routes** in the left navigation.
+3. Click **View** for the Route row.
+4. Scroll down to plugins and click **Add Plugin**.
+5. Find and select the **{{ page.name }}** plugin.
+
+    <div class="alert alert-ee blue">
+    <strong>Note:</strong> If the plugin is greyed out, then it is not available
+    for your product tier. See <a href="">Kong Gateway tiers</a>.
```

Comment:
This could also have nothing to do with product tier and more to do with security precautions. For Konnect, tranforms etc are not allowed.